src/expressions/NamedFunctionRef.ts

```diff
@@ -57,7 +56,7 @@ class NamedFunctionRef extends Expression {
 		let namespaceURI = this._functionReference.namespaceURI;
 		if (!namespaceURI) {
 			if (!this._functionReference.prefix) {
-				namespaceURI = FUNCTIONS_NAMESPACE_URI;
```

Comment:
I think the rebase did not yet really come through

Reply:
I don't understand what you mean. Why is this a problem? 

What we are doing here is that instead of using the constant, we use the variable, which is the to the same value at the constant if there is no "default function namespace" specified.

Reply:
yeah, but this was part of the default function namespace PR, right?